addons/main/functions/fnc_handleHealEh.sqf

```diff
@@ -28,6 +28,10 @@
     if ((call CBA_fnc_currentUnit) isEqualTo _unit) then {
         [_unit] call FUNC(updateHPUi);
     };
+    private _ret = [_healer] call FUNC(hasHealItems);
+    if (_ret isEqualTo 1) then {
+        _healer removeItem "FirstAidKit";
+    };
```

Comment:
Was this tested that it does not consume 2 FAKs when healing others?

Reply:
Yes, I tested it. It only consumes one regardless of healing self, others, reviving, etc. Everything seems to go through handleHealEh so it Just Works™.